packages/salesforcedx-vscode-core/src/commands/forceSourceDeployManifest.ts

```diff
@@ -49,10 +53,23 @@ export async function forceSourceDeployManifest(manifestUri: vscode.Uri) {
       return;
     }
   }
+
+  const messages: ConflictDetectionMessages = {
+    warningMessageKey: 'conflict_detect_conflicts_during_deploy',
```

Comment:
This doesn't matter right now since we don't have the Japanese translation of the message yet, but I think we should change this to:
```suggestion
    warningMessageKey: nls.localize('conflict_detect_conflicts_during_deploy'),
```

Reply:
Disregard - I see that we localize the warningMessageKey in postconditionCheckers.